packages/vite/src/node/optimizer/index.ts

```diff
@@ -310,9 +320,13 @@ export function initDepsOptimizerMetadata(
   ssr: boolean,
   timestamp?: string,
 ): DepOptimizationMetadata {
-  const hash = getDepHash(config, ssr)
+  const lockfileHash = getLockfileHash(config, ssr)
+  const configHash = getConfigHash(config, ssr)
+  const hash = lockfileHash + configHash
```

Comment:
We are missing `getHash` here
```suggestion
  const hash = getHash(lockfileHash + configHash)
```
Maybe it is better to have a composeHashes util so it is in sync with getDepHash

Reply:
Oh I wasn't mean to do another getHash here... because they are hashes already. Do we care about the length getting doubled, or could we simplify contacting them? I'd fix `getDepHash` if so

Reply:
The hash ends up used in the ?v in the optimized module id, good for it to be a hash